<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditQueryItemWidget</class>
 <widget class="QWidget" name="EditQueryItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>278</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="description_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="searchInLabel">
       <property name="text">
        <string>Search in:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="SelectBaseWidget" name="select_base_widget" native="true"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="scope_checkbox">
       <property name="text">
        <string>Recursive search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="filter_display">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_filter_button">
       <property name="text">
        <string>Edit filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectBaseWidget</class>
   <extends>QWidget</extends>
   <header>filter_widget/select_base_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name_edit</tabstop>
  <tabstop>description_edit</tabstop>
  <tabstop>scope_checkbox</tabstop>
  <tabstop>edit_filter_button</tabstop>
  <tabstop>filter_display</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
